src/styles/_catalog.scss

```diff
@@ -1,22 +1,103 @@
-.catalog {
-  display: flex;
-  flex-wrap: wrap;
+$margin-reset: 0;
```

Comment:
I think that this variable is unnecessary. If you put margin as 0 it's automatically understood that they are reset. Naming a zero - reset is unnecessary work IMHO.